kubernetes/resource_kubernetes_config_map_test.go

```diff
@@ -345,3 +378,41 @@ resource "kubernetes_config_map" "test" {
 }
 `, prefix)
 }
+
+func testAccKubernetesConfigMapConfig_binaryData(prefix string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_config_map" "test" {
+  metadata {
+    generate_name = "%s"
+  }
+
+  binary_data = {
+    one = "${filebase64("./test-fixtures/binary.data")}"
+  }
+
+  data = {
+    two = "second"
+  }
+}
+`, prefix)
+}
+
+func testAccKubernetesConfigMapConfig_binaryData2(prefix string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_config_map" "test" {
+  metadata {
+    generate_name = "%s"
+  }
+
+  binary_data = {
+    one = "${filebase64("./test-fixtures/binary2.data")}"
```

Comment:
Are the fixture files reversed (binary2.data and binary.data) on purpose here?

Reply:
Indeed :)

Reply:
Fixed.